Ginger/Ginger/Reports/HTMLReportTemplatePage.xaml.cs

```diff
@@ -92,9 +92,19 @@ public HTMLReportTemplatePage(HTMLReportConfiguration HTMLReportConfiguration)
             SetLoadedLogoImage();
             SetHTMLReportsConfigFieldsGridsView();
             SetHTMLReportsConfigFieldsGridsData(_HTMLReportConfiguration);
+            RadioButtonInit();
+        }
+        public void RadioButtonInit()
+        {
+            string currentValue = _HTMLReportConfiguration.ExecutionCalculationLevel;
+            foreach (RadioButton rdb in htmlExecutionCalculationLevelBtnsPnl.Children)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'foreach' block.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4549121)